backend/sites/src/app/services/site/site.service.ts

```diff
@@ -1719,6 +1719,21 @@ export class SiteService {
       }
 
       this.sitesLogger.log('SiteService.processSRBulkUpdates() end');
+
+      const historyLog: HistoryLog = {
+        userId: userInfo ? userInfo.sub : '',
+        content: {...site, 'isApproved': isApproved},
+        id: null,
+        whoCreated: userInfo ? userInfo.givenName : '',
+        whenCreated: new Date(),
+        whenUpdated: new Date(),
+        whoUpdated: userInfo ? userInfo.givenName : '',
+        siteId: site.siteId,
+      };
+  
+      await transactionalEntityManager.save(HistoryLog, historyLog);
+
+
```

Comment:
Is this change being made in order to pass an existing unit test?

Reply:
This was due to branch created from a previous commit, but latest dev is merged to this branch and no longer its not part of the pull request.